### PR TITLE
Add GTM

### PR DESCRIPTION
**Pull request by vandreleal**

### CODE REVIEW

File: src/environment.js [hunk 1]
```diff
@@ -2,9 +2,10 @@
 // it exposed with the define-env-vars-plugin
 
 module.exports = {
-  ALGOLIA_APP_ID: process.env.ALGOLIA_APP_ID,
   ALGOLIA_API_KEY: process.env.ALGOLIA_API_KEY,
-  WEBSITE_BASE_URL: process.env.WEBSITE_BASE_URL,
+  ALGOLIA_APP_ID: process.env.ALGOLIA_APP_ID,
   DEPLOYMENT_ENV: process.env.DEPLOYMENT_ENV,
+  GOOGLE_TAG_TRACKING_ID: process.env.DEPLOYMENT_ENV,
```
Comment (jaapbakker88) [inline src/environment.js]: should this be: 

```suggestion
  GOOGLE_TAG_TRACKING_ID: process.env.GOOGLE_TAG_TRACKING_ID,
```

**Review by jaapbakker88 — CHANGES_REQUESTED** [review]
One little comment but this should be good. Was looking with Rodolphe and we also figured out that GTM wasn't properly tracked, this should probably do it

File: src/environment.js (hunk 1 above)
Comment (vandreleal) [reply src/environment.js]: Good catch! Thanks.

**Comment by vandreleal** [comment]
> One little comment but this should be good. Was looking with Rodolphe and we also figured out that GTM wasn't properly tracked, this should probably do it 

Just applied your suggestion from the previous comment.

**Review by jaapbakker88 — APPROVED** [review]

**Pull request merged**